src/test/java/core/basesyntax/dao/StorageDaoImplTest.java

```diff
@@ -0,0 +1,83 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertSame;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class StorageDaoImplTest {
+    private static final StorageDao storageDao = new StorageDaoImpl();
+
+    @BeforeAll
+    public static void setupStorage() {
+        Storage.people.add(new User());
+        Storage.people.get(0).setLogin(null);
+        Storage.people.add(new User());
+        Storage.people.get(1).setLogin("admin");
+        Storage.people.add(null);
+    }
+
+    @Test
+    public void add_user_returnsSameUser() {
```

Comment:
![image](https://github.com/mate-academy/jv-registration-validation/assets/12864128/baeeb7c7-4f68-434f-a2f9-9815c01df146)
It is necessary to rename all methods according to convention